doc/REPOSITORY_STRUCTURE.md

```diff
@@ -66,9 +66,73 @@
 [viewer](/cloud/storage/core/libs/viewer) - Helpers for outputting some tablet-related info onto tablet monitoring pages
 
 ## Blockstore (NBS)
+[blockstore](/cloud/blockstore) - Network Block Store API and implementation - storage node (diskagentd), initiator + control plane + config plane (nbsd), client (blockstore-client), other tools and tests
+[plugin](/cloud/vm) - Legacy Network Block Store qemu plugin which can connect to NBS using either NBD or gRPC APIs
+[blockstore/config](/cloud/blockstore/config) - config proto specs
+[blockstore/public](/cloud/blockstore/public) - gRPC public API specs
+[blockstore/private](/cloud/blockstore/private) - gRPC private API specs - private API actions can be called via the ExecuteAction public API call or via blockstore-client executeaction
+[blockstore/tests](/cloud/blockstore/tests) - integration tests - most of them launch single-node BlobStorage cluster + NBS and then issue some requests and check the responses/metrics/etc
+
+### Blockstore apps
+[apps/client](/cloud/blockstore/apps/client) - cmdline client which uses mostly gRPC API - can perform almost any operation which is present in the API of nbsd
+[apps/disk_agent](/cloud/blockstore/apps/disk_agent) - storage node server
+[apps/endpoint_proxy](/cloud/blockstore/apps/endpoint_proxy) - a proxy which can maintain an NBD connection with the kernel and proxy the requests to nbsd (to allow nbsd updates without NBD device reconfiguration)
+[apps/server](/cloud/blockstore/apps/server) - the main NBS server (nbsd) - provides NBD/vhost-blk/gRPC APIs both for dataplane and controlplane ops, can host volume tablets
+[apps/server_lightweight](/cloud/blockstore/apps/server_lightweight) - nbsd version without YDB BlobStorage dependencies - uses local fs for storage, suitable for local debugging and as an nbsd mock to test your app's integration with nbsd
+[apps/tools](/cloud/blockstore/apps/tools) - misc tools for testing, debugging, log analysis and visualization
+
+### Blockstore libs
+[libs/client](/cloud/blockstore/libs/client) - client interface, gRPC client implementation, durable client (retrying client) implementation, session (remounting client) implementation, throttling client implementation
+[libs/client_rdma](/cloud/blockstore/libs/client_rdma) - client implementation over our RoCEv2-based transport lib
```

Comment:
actually there is nothing indicating we use RoCEv2 in the code so better just use `RDMA-based` here and in client